### DISCUSSION TITLE
Can you restrict the file types that can be uploaded in a blog post?

### DISCUSSION THREAD

**Discussion by jacksorjacksor** [decaporg/decap-cms#6505] [General]
A user uploaded a .jpe image file (instead of a .jpg) which stopped an article being published. Is there a way we are able to restrict this?
TY!
Rich

**Comment by bmackinney** [comment]
Have you tried using the image field's `pattern:` option? https://www.netlifycms.org/docs/widgets/#common-widget-options
([start here for patterns?](https://stackoverflow.com/questions/10473185/regex-javascript-image-file-extension)).

**Comment by jacksorjacksor** [comment]
I have not! I'll try that in the morning, thank you!

**Comment by jacksorjacksor** [comment]
Morning - @bmackinney that worked perfectly, thank you so much for the direction.

For reference, here's the regex used:

```
pattern: ['.*\.(gif|jpe?g|tiff?|png|webp|bmp)$', "Invalid file type"]
```

Thanks again!
